internal/provider/runner/runner.go

```diff
@@ -43,7 +44,7 @@ func (r *Runner) Start(ctx context.Context) (err error) {
 		if err != nil {
 			return fmt.Errorf("failed to get kubeconfig: %w", err)
 		}
-		p, err := kubernetes.New(cfg, &r.Config.Server, r.ProviderResources)
+		p, err := kubernetes.New(cfg, &r.Config.Server, r.ProviderResources, r.Elected)
```

Comment:
this won't be needed, and should be available in `r.Config.Server`

Reply:
fixed